packages/elements/src/number-field/index.ts

```diff
@@ -861,7 +859,6 @@ export class NumberField extends FormFieldElement {
       'type': 'text',
       'part': 'input',
       'inputmode': 'decimal',
-      'pattern': NUMBER_PATTERN,
```

Comment:
I don't think this change is related to the PR.

Reply:
It includes by mistake already remove it.